app/src/androidTest/java/com/example/sharingang/NotificationTest.kt

```diff
@@ -25,13 +37,52 @@ class NotificationTest {
 
     @Test
     fun aNotificationCanBeReceived() {
+        val itemTitle = "T"
+
+        navigate_to(R.id.newEditFragment)
+        Espresso.onView(ViewMatchers.withId(R.id.itemTitle)).perform(
+            ViewActions.typeText(itemTitle),
+            ViewActions.closeSoftKeyboard()
+        )
+        val buttonCreate = Espresso.onView(ViewMatchers.withId(R.id.createItemButton))
+        buttonCreate.check(ViewAssertions.matches(ViewMatchers.withText("Create Item")))
+        buttonCreate.perform(ViewActions.click())
```

Comment:
I've merged createItemButton and editItemButton into a single saveItemButton so you'll need to change it when merging.
Also with the updated UI, it no longer fits on the emulator screen you we have to scroll to it before clicking
```suggestion
        val buttonCreate = Espresso.onView(ViewMatchers.withId(R.id.saveItemButton))
        buttonCreate.check(ViewAssertions.matches(ViewMatchers.withText("Create Item")))
        buttonCreate.perform(ViewActions.scrollTo(), ViewActions.click())
```